INSTALL.md

```diff
@@ -331,6 +354,11 @@ Specifies if the metis version is 5 or greater (OFF by default).
 Root directory for Metis library
 
 #### Trilinos
+On Ubuntu 18.04, the following command installs the necessary files:
+
+```Shell
+sudo apt-get install trilinos-all-dev
```

Comment:
I would add the same for the Metis (Check the CI-DockerFile for the exact command)